doc/rst/source/supplements/mgd77/mgd77magref.rst

```diff
@@ -63,7 +63,7 @@ Optional Arguments
     from:ftp://ftp.ngdc.noaa.gov/STP/GEOMAGNETIC_DATA/INDICES/DST/ (the
     *Dst* coefficients) and
     http://umbra.nascom.nasa.gov/sdb/yohkoh/ys_dbase/indices_flux_raw/Penticton_Absolute/monthly/MONTHPLT.ABS
-    (The *F10.7* index file is a MONTHPLT.ABS). NOTE: since the *Dst* files
+    (The *F10.7* index file is a MONTHPLT.ABS). **Note**: since the *Dst* files
```

Comment:
Also for "note"?

Reply:
Yes, we use **Note**: exclusively elsewhere.

Reply:
Ok.